packages/dev/core/src/Misc/greasedLineTools.ts

```diff
@@ -0,0 +1,156 @@
+import { VertexBuffer } from "./../Buffers/buffer";
+import { Vector3 } from "../Maths/math.vector";
+import type { AbstractMesh } from "../Meshes/abstractMesh";
+
+export namespace GreasedLineTools {

```

Comment:
no namespaces :-)


I understand what you are trying to do here, but namespaces make our definitions a little chaotic.
We usually solve it using a class that contains static members only.

Reply:
Yes, actually I planned to move the tools to static methods of a class. Will this work for you?